smallrye-reactive-messaging-in-memory/src/main/java/io/smallrye/reactive/messaging/connectors/i18n/InMemoryLogging.java

```diff
@@ -0,0 +1,12 @@
+package io.smallrye.reactive.messaging.connectors.i18n;
+
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.MessageLogger;
+
+@MessageLogger(projectCode = "SRRML")
+public interface InMemoryLogging {
+
+    InMemoryLogging log = Logger.getMessageLogger(InMemoryLogging.class, "io.smallrye.reactive.messaging");
+
+    // 19667-19999 (logging)
```

Comment:
Shouldn't it be first? It's above in the other connectors.

Reply:
I started from the bottom (for connectors). So the implementation is first, then there is a gap (for future use, either impl from the one side or new connectors from the other side. That was my thinking.